### PR TITLE
Update quick-install.md

### PR DESCRIPTION
**Pull request by galpizarsfx**
This is testing the Smart Agent tile to see if we can manipulate text that appears both in the UI and in docs.signalfx. This needs to be merged so that we can see the results in lab. Afterwards, whether or not the test is successful, this will need to be unmerge so that in another PR, I will make the official change, along with some other updates.

### CODE REVIEW

File: docs/quick-install.md [hunk 1]
```diff
@@ -70,17 +70,16 @@ _Note: if you have previously configured another metric collection agent on your
...
-- Determine YOUR\_SIGNAL_FX_REALM from your [profile page](https://docs.signalfx.com/en/latest/getting-started/get-around-ui.html#user-profile-avatar-and-color-theme) in the SignalFx web application.
-
 To install the Smart Agent on a single Linux host, enter:
 
-```sh
-curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
+```sh curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
 sudo sh /tmp/signalfx-agent.sh --realm YOUR_SIGNALFX_REALM YOUR_SIGNALFX_API_TOKEN
+
+<!-- In the above command, YOUR_SIGNALFX_REALM represents the realm that is shown on your profile page
```
Comment (keitwb) [inline docs/quick-install.md]: I don't think you need to make this a comment, just make it regular markdown.

**Comment by galpizarsfx** [comment]
I thought the comment would help to manipulate the appearance of the text
between the tile and the doc site. In other words,

* In the tile, the text would display the user's specific realm and token.
* In the doc site, the same text would literally display
YOUR_SIGNALFX_REALM.

Does that make sense?

George

**Comment by keitwb** [comment]
No, it will just make it disappear everywhere if it is a comment.

**Comment by galpizarsfx** [comment]
Okay, then in that case, we can simply ignore and close this request. As an FYI, in the near future, I'll be sending another PR which will reorganize the Smart Agent content with some editing as well.

**Pull request closed without merging**